client/src/components/ChatMessage.tsx

```diff
@@ -66,16 +75,22 @@ export default function ChatElement(props: {message: ChatMessage}): ReactElement
             }
     }
 
-    return <StyledText className={"chat-message " + style}>{translateChatMessage(message)}</StyledText>;
+    return <StyledText className={"chat-message " + style}>{translateChatMessage(message, playerNames)}</StyledText>;
 }
 
-function playerListToString(playerList: PlayerIndex[]): string {
+export function getPlayerNames(): string[] {
+    switch (GAME_MANAGER.state.stateType) {
+        case "game":
+            return GAME_MANAGER.state.players.map((player) => player.toString());
+        default:
+            return [];
+    }
+}
+
+function playerListToString(playerList: PlayerIndex[], playerNames: string[]): string {
```

Comment:
nit: this function doesn't take a player list as input, should rename?

Reply:
`playersToString`?

Reply:
`translatePlayers`?

Reply:
That is a list of players